app/components/metadata-form/author-form/component.js

```diff
@@ -0,0 +1,41 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import Changeset from 'ember-changeset';
+import lookupValidator from 'ember-changeset/utils/validator-lookup';
+import AuthorValidations from '../../../validations/author';
```

Comment:
We don't have a style rule or anything, nor is my opinion very strong about this point, but I tend to lean towards using absolute paths for local imports for readability 

```
import 'jhu-deposit-ui/validations/author';
```

Reply:
Good call